src/components/Modals/Onboarding/OnboardingTransferExamPropertyDropdown.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div
+    :class="columnWide ? 'onboarding-select--columnWide' : 'onboarding-select--column'"
+    :style="{ borderColor: boxBorder }"
+    v-click-outside="closeDropdownIfOpen"
+  >
+    <label class="onboarding-label">{{ propertyName }}</label>
+    <div class="onboarding-select onboarding-input">
+      <div
+        class="onboarding-dropdown-placeholder college-major-minor-wrapper"
+        @click="showHideContent()"
+      >
+        <div
+          class="onboarding-dropdown-placeholder college-major-minor-placeholder"
+          :style="{ color: placeholderColor }"
+        >
+          {{ choice }}
+        </div>
+        <div
+          class="onboarding-dropdown-placeholder college-arrow"
+          :style="{ borderTopColor: arrowColor }"
+        ></div>
+      </div>
+      <div class="onboarding-dropdown-content college-content" v-if="shown">
+        <div
+          v-for="(choice, index) in availableOptions"
+          :key="index"
+          class="onboarding-dropdown-content-item"
+          @click="selectChoice(choice)"
+        >
+          {{ choice }}
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue, { PropType } from 'vue';
+import NewCourse from '@/components/Modals/NewCourse/NewCourse.vue';
+import { clickOutside } from '@/utilities';
+// @ts-expect-error: typescript cannot understand scss variable imports.
+import { inactiveGray, yuxuanBlue, lightPlaceholderGray } from '@/assets/scss/_variables.scss';
+
+Vue.component('newCourse', NewCourse);
+
+export default Vue.extend({
+  props: {
+    propertyName: { type: String, required: true },
+    columnWide: { type: Boolean, required: true },
+    availableOptions: { type: Array as PropType<readonly unknown[]>, required: true },
+    choice: { type: [String, Number], required: true },
+  },
+  data() {
+    return {
+      shown: false,
+      stopClose: false,
+      boxBorder: '',
+      arrowColor: '',
+      placeholderColor: '',
```

Comment:
Can we have the `placeholderColor` be `lightPlaceholderGray` if the placeholder is not "Select one" but is a previously inputted exam? The right (master) panel of the screenshot shows what we would want if the user had just opened up onboarding and saw the AP exams that they'd inputted before. 
![Screen Shot 2021-02-01 at 2 14 10 PM](https://user-images.githubusercontent.com/54298311/106506460-c30a3f00-6497-11eb-8ee8-7e1a1e67403b.png)